_drafts/2017-05-31-product-goals.md

```diff
@@ -0,0 +1,53 @@
+---
+title:  "Product goals for 2017"
+author: borsi
+tags: [goals, community, roadmap, redesign, statistics]
+---
+## Everybody needs goals
+
+Us, too. In fact we need better goals, than *"making more money"* or *"creating better stuff"*. So last year we decided to be [SMART][bb699076] about it and set high-level product goals for ourselves and for our community. After all, we want to be the best ECM development platform for a large community of developers. But how do we get there?
```

Comment:
a smart-t is megszínesíteném, [például ezzel](https://www.google.hu/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwjD8NCz06vUAhVBPRoKHX1bB-gQjRwIBw&url=http%3A%2F%2Fwww.j6design.com.au%2Fsetting-smart-goals%2F&psig=AFQjCNGkZnQ2CgvdFgHxO-GzragYwLf0aA&ust=1496921894032544)